### DISCUSSION TITLE
Can the Octoprint settings be added to the Wiki?

### DISCUSSION THREAD

**Discussion by joeclifford** [Jyers/Marlin#294] [Ideas]
I've had to look through past discussions and closed issues to find the needed Octoprint settings for the LCD to display additional information during prints.

I found most the settings in a non-default branch.

https://github.com/Jyers/Marlin/blob/Ender_3_V2_Rewrite/Images/Octoprint%20Settings.JPG

I had to look in the comments for a closed issue to find the needed settings for "Added Support for Displaying Filename During Host Print" that was added in the 1.2.2 release.

#238 
https://github.com/Jyers/Marlin/issues/238#issuecomment-798940245

It would be good if these settings could be added to a section in the Wiki.

**Comment by Jyers** [comment]
Yes, I realized that this should be a wiki page after adding the filename functionality, but haven't gotten around to creating the page yet. I'll definitely make it soon.
